elections/admin.py

```diff
@@ -27,19 +28,43 @@ def proc_row(row):
     return response 
 download_voters.short_description = "Download selected voters"
 
+class BallotInline(admin.TabularInline):
+    model = Ballot
+    form = BallotForm
+    extra = 0
+    show_change_link = True
+    can_delete = False
+    exclude = ("introduction", )
+
 class ElectionAdmin(admin.ModelAdmin):
     list_display = ("__str__", "nomination_start", "vote_start")
+    inlines = (BallotInline, )
+
+class FilterByElection(admin.ModelAdmin):
+    list_filter = ("election", )
 
 class BallotAdmin(admin.ModelAdmin):
+    form = BallotForm
     list_display = ("__str__", "position_number", "election")
     list_filter = ("election", )
 
+    def get_form(self, request, obj=None, **kwargs):
+        form = super(BallotAdmin, self).get_form(request, obj, **kwargs)
+        form.base_fields['election'].initial = get_current_election()
+        return form
+
 class VoterAdmin(admin.ModelAdmin):
-    list_filter = ("election", )
     actions = (download_voters,)
+    list_filter = ("election", "has_voted")
+    list_display = ("__str__", "has_voted")
+    search_fields = ("user__username", "user__first_name", "user__last_name", "user__email")
+
+class CandidateAdmin(admin.ModelAdmin):
+    list_display = ("__str__", "ballot", "incumbent")
+    list_filter = ("ballot__election",)
 
 admin.site.register(Election, ElectionAdmin)
 admin.site.register(Ballot, BallotAdmin)
 admin.site.register(Voter, VoterAdmin)
-admin.site.register(Nominee)
-admin.site.register(Candidate)
\ No newline at end of file
+admin.site.register(Candidate, CandidateAdmin)
```

Comment:
nominee = candidate now? from an admin perspective at least?

Reply:
Nope, two separate tables. I removed anonymous stuff from admin like Nominees and votes. They are now compiled into a ballot's admin.

Reply:
This is where nominees get folded into the Ballot admin display (via its form): https://github.com/iQuISE/iquise-website/pull/49/files#diff-3e2c52ed0a68b670e1e607796872a1a6d30512a36149f39a09c15940c5f2b842R17